### ISSUE TITLE
reveal hand result not calculated automatically when players reveal simultaneously

### ISSUE THREAD

**Issue by mysticpickle** [db0/Doomtown-Reloaded-OCTGN#32]
I've noticed that when both players ctrl-Q lowball close to the same time or reveal shootout hands close to the same time, the result of the hands will display an error: "inconsistent state. marker count invalid" and the result will not be calculated in the log.

![image](https://cloud.githubusercontent.com/assets/3962478/5750265/257b61e2-9c07-11e4-86e9-7a563df56336.png)

**Comment by swider33** [comment]
The same happens on shootout hand reveal, and in that case the rank of one player hand is taken from previous reveal of shootout hand for comparison. For instance player 1 reveals hand of strength 5 and player 2 reveals hand of strength 4, if the bug happens and we reveal next shootout hands, lets say both reveal hands of strength 7, in the log communicate will say that player 1 won by 3  or player 2 won by 2.

**Comment by db0** [comment]
yeah this is not really possible to fix. I suggest you always wait for one side to reveal their hand before the other does.

**Comment by herrderzeit** [comment]
Couldn't you add a script that brings up a dialog box on both sides asking if both players are ready to reveal their hands instead or a ready button of some sort? When both players click ready it reveals both hands. Is that possible?

**Comment by mysticpickle** [comment]
That sounds like a pretty good idea!

**Comment by db0** [comment]
Unfortunately it's not possible. I do have a workaround planned however. I just need to implement it

**Comment by mysticpickle** [comment]
Darn, well it's just a minor inconvenience right now.  Thanks for looking into it db0! :D
